src/ocamlorg_web/lib/router.ml

```diff
@@ -23,66 +23,81 @@ let media_loader =
       let* store in
       Media.last_modified store (Mirage_kv.Key.v path))
 
+let head_handler handler request =
+  let open Lwt.Syntax in
+  let* response = handler request in
+  let* body = Dream.body response in
+  let length = body |> String.length |> string_of_int in
+  Dream.drop_header response "Content-Length";
```

Comment:
Actually, I haven't observed a Content-Length here. I fail to understand why yet, intuitively, it should be there.

Reply:
If I understand correctly, content-length is added by Dream **if it's not already present**. So you won't observe a Content-Length at the middleware level unless the handler explicitly sets it. I'm not sure we do this at the moment, but I can see a few use cases where it would be useful, for instance when streaming a static file.